<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>941</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #26a0da, 		stop:1 #588eb0);
	border-radius: 20px
}

#btn_testP1, #btn_testP2, #btn_testP3 , #btn_testP4, #btn_testP5, #btn_testP6{
	color: black; 
	background-color: rgb(255, 204, 0);
	border-radius: 2px;
	padding: 10px;
	font-weight: bold;
}

#btn_testP1:checked{
	background-color: rgb(0, 170, 0);
}
#btn_testP2:checked{
	background-color: rgb(0, 170, 0);
}
#btn_testP3:checked{
	background-color: rgb(0, 170, 0);
}
#btn_testP4:checked{
	background-color: rgb(0, 170, 0);
}
#btn_testP5:checked{
	background-color: rgb(0, 170, 0);
}
#btn_testP6:checked{
	background-color: rgb(0, 170, 0);
}


#btn_salir{
	background-color:#414345;
	border-radius:5px;
	border:1px solid #337bc4;
	color:white;
	font-weight:bold;
	text-decoration:none;
}

#btn_salir:hover {
	background-color:#757F9A
}
#btn_salir:active {
	position:relative;
	top:1px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>171</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_testP1">
       <property name="text">
        <string>Test Panel 1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_testP2">
       <property name="text">
        <string>Test Panel 2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_testP3">
       <property name="text">
        <string>Test Panel 3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_testP4">
       <property name="text">
        <string>Test Panel 4</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_testP5">
       <property name="text">
        <string>Test Panel 5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_testP6">
       <property name="text">
        <string>Test Panel 6</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="terminalPanel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>561</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;
padding: 5px</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_testear">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
color: white;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Comenzar Tester</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stopTest">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
color: white;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Interrumpir</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;
padding-left: 5px;
font-weight: bold;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TESTEAR PANELES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salir">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_salir</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>801</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>944</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
